.travis/deploy.sh

```diff
@@ -1,9 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
I'm not sure that we should delete this yet. Not sure if it is still being used to cut releases. I think we should wait until a full replacement is in place before deleting.

Reply:
I think this is safe to remove as the reference to this script is commented out:

https://github.com/geotrellis/maml/blob/1d86b67ef0dd7f77e0ab46f1664b8297b0c502dd/.travis.yml#L12-L26

I don't think it's being executed ad hoc either, as it references Travis-specific environment variables.

Reply:
👍